tests/split_jaws.py

```diff
@@ -0,0 +1,129 @@
+import unittest
+import os
+
+from utils.channel_access import ChannelAccess
+from utils.ioc_launcher import IOCRegister, get_default_ioc_dir
+from utils.test_modes import TestModes
+from utils.testing import parameterized_list
+from parameterized import parameterized
+
+MTR_01 = "GALIL_01"
+MTR_02 = "GALIL_02"
+
+# PV names for motors
+JAWS_BASE_PV = "MOT:JAWS1"
+DIRECTIONS = ["N", "S", "E", "W"]
+
+# PV names for GALIL motors
+UNDERLYING_MTR_NORTH = "MOT:MTR0101"
+UNDERLYING_MTR_SOUTH = "MOT:MTR0102"
+UNDERLYING_MTR_EAST = "MOT:MTR0201"
+UNDERLYING_MTR_WEST = "MOT:MTR0202"
+
+UNDERLYING_AXES = [UNDERLYING_MTR_NORTH, UNDERLYING_MTR_SOUTH, UNDERLYING_MTR_EAST, UNDERLYING_MTR_WEST]
+
+# Tests will fail if JAWS support module is not up to date and built
+test_path = os.path.abspath(os.path.join(os.path.dirname(os.path.dirname(__file__)), "test_config", "motor_test"))
+
+IOCS = [
+    {
+        "name": MTR_01,
+        "directory": get_default_ioc_dir("GALIL"),
+        "pv_for_existence": "AXIS1",
+        "macros": {
+            "MTRCTRL": "01",
+            "GALILCONFIGDIR": test_path.replace("\\", "/"),
+        },
+    },
+    {
+        "name": MTR_02,
+        "directory": get_default_ioc_dir("GALIL", iocnum=2),
+        "pv_for_existence": "AXIS1",
+        "macros": {
+            "MTRCTRL": "02",
+            "GALILCONFIGDIR": test_path.replace("\\", "/"),
+        },
+    },
+]
+
+
+TEST_MODES = [TestModes.DEVSIM]
+
+TEST_POSITIONS = [-5, 0, 10, 10e-1]
+
+
+class SplitJawsTests(unittest.TestCase):
+
+    """
+    Tests for vertical jaws
+    """
+    def setUp(self):
+        self._ioc = IOCRegister.get_running("jaws")
+        self.ca = ChannelAccess(default_timeout=30)
+        for axis in UNDERLYING_AXES:
+            self.ca.set_pv_value("{}.VMAX".format(axis), 100)
+            self.ca.set_pv_value("{}.VELO".format(axis), 100)
+
+    def test_GIVEN_ioc_started_THEN_north_underlying_mtr_fields_can_be_read(self):
```

Comment:
The first four tests can easily be paramterized.

Reply:
With the latest changes, there are 2 test classes which both run the same set of tests for regular and split jaws respectively. With parameterized tests, I could not overwrite the underlying motor PV addresses in the split jaws test class, so I have opted against doing this.